src/main/io/ledstrip.c

```diff
@@ -948,10 +946,10 @@ void ledStripUpdate(uint32_t currentTime)
         // sanitize timer value, so that it can be safely incremented. Handles inital timerVal value.
         // max delay is limited to 5s
         int32_t delta = cmp32(now, timerVal[timId]);
-        if (delta < 0 && delta > -LED_STRIP_MS(5000))
+        if (delta < 0 && delta > -HZ_TO_MICROS(0.2f))
```

Comment:
Max delay shall be 5s, not 0.2Hz ... Sometimes it's easier to thing in time units ...